/**:
  ros__parameters:
    w_rot: -0.02191253424080549
    x_rot: 0.7223774484886832
    x_trans: 0.00955877856225443
    y_rot: 0.6910410169237099
    y_trans: -0.5546086937732368
    z_rot: -0.012368334049918495
    z_trans: 0.7865807789230761
